app/presenters/tree_node_builder.rb

```diff
@@ -44,16 +44,20 @@ def build_id
     object.kind_of?(Hash) ? build_hash_id : build_object_id
   end
 
-  def build
-    case object
-    when AssignedServerRole   then assigned_server_role_node(object)
-    when AvailabilityZone     then generic_node(object.name,
+  # FIXME: This is a rubocop disaster... fixed the alignment with the params,
+  # but that is about as far as I was willing to go with this one...
```

Comment:
Arguably a `WONTFIX` as well, but basically this will require a major re-thinking of how this hash is structured, possibly adding methods to reduce the line length and the nasty procs required here.  Aligning the params was easy, but beyond that the refactoring required to make this pass rubocop normally was out of scope for this PR.  Disabling these cops for the hash instantiation is just the easiest way for now to avoid about 80 cops from blasting this PR, and future PRs that modify this hash.